org.mixedrealitytoolkit.input/Tracking/TrackedPoseDriverWithFallback.cs

```diff
@@ -0,0 +1,168 @@
+        /// The fallback Input System action to use for Position Tracking for this GameObject when the default position
+        /// input action has no data. Must be a Vector3Control Control.
+        /// </summary>
+        public InputActionProperty FallbackPositionAction => fallbackPositionAction;
+
+        [SerializeField, Tooltip("The fallback Input System action to use for Rotation Tracking for this GameObject when the default rotation input action has no data. Must be a Vector3Control Control.")]
+        private InputActionProperty fallbackRotationAction;
+
+        /// <summary>
+        /// The fallback Input System action to use for Rotation Tracking for this GameObject when the default rotation
+        /// input action has no data. Must be a Vector3Control Control.
+        /// </summary>
+        public InputActionProperty FallbackRotationAction => fallbackRotationAction;
+
+        [SerializeField, Tooltip("The fallback Input System action to get the Tracking State for this GameObject when the default track status action has no data. If not specified, this will fallback to the device's tracking state that drives the position or rotation action. Must be a IntegerControl Control.")]
+        private InputActionProperty fallbackTrackingStateAction;
+
+        /// <summary>
+        /// The fallback Input System action to get the Tracking State for this GameObject when the default track status
+        /// action has no data. If not specified, this will fallback to the device's tracking state that drives the position
+        /// or rotation action. Must be a IntegerControl Control.
+        /// </summary>
+        public InputActionProperty FallbackTrackingStateAction => fallbackTrackingStateAction;
+
+        #endregion Fallback action values
+
+        #region TrackedPoseDriver Overrides 
+        /// <inheritdoc />
+        protected override void PerformUpdate()
+        {
+            base.PerformUpdate();
+
+            if (trackingStateInput.action == null)
+            {
+                Debug.LogWarning("TrackedPoseDriverWithFallback.trackingStateInput.action is null, no fallback will be used.");
+                return;
+            }
+
+            var hasPositionAction = positionAction != null;
+            var hasPositionFallbackAction = fallbackPositionAction != null;
+
+            var hasRotationAction = rotationAction != null;
+            var hasRotationFallbackAction = fallbackRotationAction != null;
+
+            InputTrackingState inputTrackingState = (InputTrackingState)trackingStateInput.action.ReadValue<int>();
+            InputTrackingState fallbackInputTrackingState = InputTrackingState.None;
+
+            // If default InputTrackingState does not have position and rotation data, use fallback if it exists
+            if (!inputTrackingState.HasFlag(InputTrackingState.Position) && !inputTrackingState.HasFlag(InputTrackingState.Rotation) && FallbackTrackingStateAction.action != null)
```

Comment:
I don't think you need lines 106 -109.

Why are you overriding `inputTrackingState` with `FallbackTrackingStateAction`?

If inputTrackingState starts out with no valid position or rotation, then we want to use the  fallback.  However, if you're setting `inputTrackingState` to `FallbackTrackingStateAction`, then lines 123 - 124 and 130-131 will never execute.

Reply:
> I don't think you need lines 106 -109.
> 
> Why are you overriding `inputTrackingState` with `FallbackTrackingStateAction`?
> 

I added a comment explaining why.
<img width="403" alt="image" src="https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/84108471/8f7050f2-326d-4760-acc1-fc63c4a750c3">

> If inputTrackingState starts out with no valid position or rotation, then we want to use the fallback. However, if you're setting `inputTrackingState` to `FallbackTrackingStateAction`, then lines 123 - 124 and 130-131 will never execute.

Reply:
done in https://github.com/ms-RistoRK/MixedRealityToolkit-Unity/commit/c902133124685769d3780be832859ae1414c21d1 , resolving.